contracts/BSCValidatorSet.sol

```diff
@@ -69,11 +69,18 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   // Corresponds strictly to currentValidatorSet
   // validatorExtraSet[index] = the `ValidatorExtra` info of currentValidatorSet[index]
   ValidatorExtra[] public validatorExtraSet;
+
   // BEP-131 candidate validator
   uint256 public numOfCabinets;
   uint256 public maxNumOfCandidates;
   uint256 public maxNumOfWorkingCandidates;
 
+  // BEP-126 Fast Finality
+  uint256 public FINALITY_REWARD_RATIO = 10;
+
+  uint256 public finalityRewardRatio;
+  bool public finalityUpdated;
```

Comment:
For this two newly added fields, the default of them is zero after fork, please ensure it is compatible and expected.

Reply:
Fixed